internal/tools/orchestrator/endpoints/instance.go

```diff
@@ -315,9 +315,18 @@ func (e *Endpoints) getPodStatusFromK8s(runtimeID, serviceName string) ([]apistr
 			continue
 		}
 
+		// TODO change `DICE_CLUSTER_NAME` to  `core.erda.cloud/cluster-name`
 		clusterName := ""
-		if _, ok := pod.Labels["DICE_CLUSTER_NAME"]; ok {
-			clusterName = pod.Labels["DICE_CLUSTER_NAME"]
+		for _, container := range pod.Spec.Containers {
+			if clusterName != "" {
+				break
+			}
+			for _, env := range container.Env {
+				if env.Name == "DICE_CLUSTER_NAME" {
```

Comment:
you'd better use `apistructs.DICE_CLUSTER_NAME`.
literal string is not recommended.